src/System.Private.CoreLib/shared/System/Threading/Semaphore.cs

```diff
@@ -41,19 +33,13 @@ public Semaphore(int initialCount, int maximumCount, string name, out bool creat
         private static void VerifyCounts(int initialCount, int maximumCount)
         {
             if (initialCount < 0)
-            {
-                throw new ArgumentOutOfRangeException(nameof(initialCount), SR.ArgumentOutOfRange_NeedNonNegNumRequired);
```

Comment:
Could you please delete `ArgumentOutOfRange_NeedNonNegNumRequired` from the resource file as well? It looks unused after this change.